python/ray/includes/common.pxd

```diff
@@ -373,7 +373,43 @@ cdef extern from "ray/gcs/pubsub/gcs_pub_sub.h" nogil:
 
         CRayStatus PublishFunctionKey(const CPythonFunction& python_function)
 
+    cdef cppclass CPythonGcsSubscriber "ray::gcs::PythonGcsSubscriber":
+
+        CPythonGcsSubscriber(
+            const c_string& gcs_address, CChannelType channel_type,
+            const c_string& subscriber_id, const c_string& worker_id)
+
+        CRayStatus Connect()
+
+        CRayStatus Subscribe()
+
+        int64_t last_batch_size()
+
+        CRayStatus PollError(
+            c_string* key_id, int64_t timeout_ms, CErrorTableData* data)
```

Comment:
Ok yeah ideally returning StatusOr<Struct> instead of outparams would be great but maybe not a priority

Reply:
Yeah, that's **certainly** out of scope for this PR